### ISSUE TITLE
Using square brackets in only_if

### ISSUE THREAD

**Issue by csandanov** [walter-cd/walter#156]
I'm trying to use following condition `only_if: [ "$MY_ENV_VAR" = "value" ]` and get 
```
ERROR yaml: line 4: did not find expected alphabetic or numeric character
ERROR failed to create Walter
```
Works fine with `only_if: test "$MY_ENV_VAR" = "value"`. Is it how walter suppose to work (no square brackets support)?

**Comment by bkcsoft** [comment]
Try putting the square brackets in quotes, like so `only_if: "[ \" $FOO\"
== \"bar\" ]"`

**Comment by takahi-i** [comment]
Thank you very much for the valuable comment @csandanov. Thank you so much for showing us the workaround.  I made the patch containing the description on the workaround and the test case in #157.

**Comment by takahi-i** [comment]
Fixed with  #157.
